scania-digital-design-system/tegel
# test(block): added unit test

## **Describe pull-request**  
Adding unit test for Block component.

## **Issue Linking:**  
- **Jira:** Add ticket number after `CDEP-`: [CDEP-3024](https://tegel.atlassian.net/browse/CDEP-3024)    

## **How to test**  
1. Open a branch on your local.
2. Try to run `npm run test`
3. All tests should pass
4. Take a look at block component screenshots
5. Try to change something in a test or nested level
6. Run tests again
7. The test should fail

## **Checklist before submission**

## **Additional context**  
This is first component to have dark mode test ;) 


[CDEP-3024]: https://tegel.atlassian.net/browse/CDEP-3024?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
